lib/pbench/server/database/models/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""
+For any new database model added in the models directory
+an import statemnt of the same is required here.
+"""
+from pbench.server.database.models.active_tokens import ActiveTokens  # noqa
+from pbench.server.database.models.users import User  # noqa
+from pbench.server.database.models.tracker import Dataset, Metadata  # noqa
```

Comment:
Are the `# noqa` directives required, here?  If so, can you add the code(s) for the diagnostics that we want to suppress?